cl/lib/storage.py

```diff
@@ -104,3 +104,7 @@ def get_available_name(
 
 class SubDirectoryS3ManifestStaticStorage(S3ManifestStaticStorage):
     location = "static"
+
+
+class RecapEmailSESStorage(S3Boto3Storage):
+    bucket_name = "free-law-project-pacer-emails-development"
```

Comment:
This is the bucket name S3 where SES-formatted emails sit after they're received and until they're processed by the celery task, right? If so, maybe a better name would be `global-recap-email-inbox`? Might be worth a slack chat to find the best value here, since I suspect it's utterly unchangeable later.

Reply:
Yes, good point. Open to anything you suggest.

Reply:
OK my vote, then, is: `recap-dot-email-global-inbox`